support/util/util.go

```diff
@@ -318,3 +319,13 @@ func ApplyAWSLoadBalancerSubnetsAnnotation(svc *corev1.Service, hcp *hyperv1.Hos
 		svc.Annotations["service.beta.kubernetes.io/aws-load-balancer-subnets"] = subnets
 	}
 }
+
+func DoesMgmtClusterAndNodePoolCPUArchMatch(nodePoolArch string) error {
```

Comment:
It's there any unit test for this?

Reply:
nah nvm, in order to simulate different arches we will need to do the unit tests for each arch specifically.